apps/aecore/src/aec_vm_chain.erl

```diff
@@ -423,7 +459,8 @@ call_contract(Target, Gas, Value, CallData, CallStack,
                                     call_data   => CallData,
                                     call_stack  => CallStack }),
             case apply_transaction(CallTx, State) of
```

Comment:
Does this lose the `off_chain` from the environment? (See also https://github.com/aeternity/epoch/pull/1592/files?utf8=%E2%9C%93&diff=split&w=1#r222327210 ) What is the impact?

Reply:
The `off_chain` is kept in the `State`